### DISCUSSION TITLE
Add a drop down for purchase amount

### DISCUSSION THREAD

**Discussion by Skadis-Venom** [gameserverapp/Platform#918] [Suggestions]
When players are purchasing with tokens, adding a drop down or a box to add an amount would be nice. For instance, I have bushberry seeds in my shop. Instead of buying it once, there would be an option next to the player name that would allow the player to purchase multiple at once. As it sits right now, We click to purchase, and then the page reloads confirming the purchase, then we have to repeat that process for multiple of one item.

**Comment by babsgosgens** [comment]
Same as #654 ?

**Reply by mvaessen** [reply]
Good catch MG

**Comment by GeckoGamer** [comment]
I agree a dropdown menu would be good for purchase amounts, would save space on the web page for users and make it look cleaner, for example, when they buy tokens, the selection is just "tokens" then they can select from amounts available via the dropdown, or if someone wants to buy 10 dinos for Ark, they can select 10 from a drop down. It is similar to #654, but I think drop down will be easier, with a pre-determined selection.

**Comment by Buck-Irl** [comment]


**Reply by goosenation** [reply]
You’ll need to make a separate suggestion for that.

**Reply by Buck-Irl** [reply]
its kinda same, same, but different no? lol

**Reply by goosenation** [reply]
I’d argue its different. :)
It’s an intersting one for sure, variations on the same shoppack with different arguments. Different featurethough.

**Reply by Buck-Irl** [reply]
ok, I shall concede and make a new suggestion. thanks again MG
mine could be reverse engineered for quantity also tho, just saying lol

**Reply by mvaessen** [reply]
Thanks MG, i agree this belongs in a separate item

**Comment by mvaessen** [comment]
This feature is now available, thanks for the suggestion!
